recursion/core/src/poseidon2_wide/external.rs

```diff
@@ -29,7 +36,38 @@ pub struct Poseidon2WideChip;
 #[derive(AlignedBorrow, Clone, Copy)]
 #[repr(C)]
 pub struct Poseidon2WideCols<T> {
-    pub dummy_cols: [T; 160],
+    pub input: [T; WIDTH],
+    pub output: [T; WIDTH],
+    external_rounds: [Poseidon2WideExternalRoundCols<T>; NUM_EXTERNAL_ROUNDS],
+    internal_round_chunks: [Poseidon2WideInternalRoundChunkCols<T>; NUM_INTERNAL_ROUND_CHUNKS],
+    last_internal_rounds: [Poseidon2WideLastInternalRoundCols<T>; NUM_LAST_INTERNAL_ROUNDS],
+}
+
+// Columns required for external rounds
```

Comment:
nit: can you follow the commenting style of the rest of the repo?

you can look here https://github.com/succinctlabs/sp1/blob/main/recursion/core/src/poseidon2/external.rs

In general, full sentences, proper capitalization periods.

Reply:
btw, use /// for comments in rust for structs/functions